cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/containers/NameMappingsTest.java

```diff
@@ -249,7 +249,7 @@
             + "               \"newServicePath\": \"/new_default\","
```

Comment:
I see some little adaptations of existing test files, which I understand are not covering the new functionality.

Thus, maybe some additional test should be added to cover the new functionality?

Reply:
I agree with @fgalan

Reply:
Then @joelcamus  please, could you provide some tests to cover new functionality ?